TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -1059,6 +1070,13 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 		}.property('features.resize', '_originalImageUri'),
 
 		/**
+		 * Swaps the small preview inspector image with the original for cropping preview
+		 */
+		_swapPreviewWithOriginal: function() {
+				this.set('_inspectorImageUri', this.get('_originalImageUri'));
```

Comment:
to keep it consistent I'd suggest to rename `_inspectorImageUri` to `_thumbnailImageUri`